pyspider/database/elasticsearch/resultdb.py

```diff
@@ -18,9 +18,9 @@ class ResultDB(BaseResultDB):
 
     def __init__(self, hosts, index='pyspider'):
         self.index = index
-        self.es = Elasticsearch(hosts=hosts)
+        self.es: Elasticsearch = Elasticsearch(hosts=hosts)
 
-        self.es.indices.create(index=self.index, ignore=400)
```

Comment:
Wouldn't this cause IndexAlreadyExistsException error when the resultdb are already exists?